packages/replayer/src/assets/snapshot-assets.ts

```diff
@@ -48,13 +94,16 @@ export const snapshotAssets = async (opts: {
 // a little complexity on the server side (e.g. we'd need to save a mappings.json file that
 // maps from exact URL to the contents of returned resource, and serve requests based on this,
 // and we'd lose the debuggability of a simple folder structure).
-function withoutQueryParams(url: string) {
+const withoutQueryParams: (url: string) => string = (url) => {
   const parsed = new URL(url);
   parsed.search = "";
   return parsed.toString();
-}
+};
 
-function getFilePath(trimmedUrl: string, contentType: string | undefined) {
+const getFilePath: (
+  trimmedUrl: string,
+  contentType: string | undefined
```

Comment:
```suggestion
  contentType: string
```
to avoid `undefined`.